static/css/avatar-reaction-bubble.css

```
@@ -0,0 +1,200 @@
+    left: 52%;
+}
+
+#avatar-reaction-bubble .avatar-reaction-bubble-content {
+    position: relative;
+    z-index: 1;
+    display: inline-block;
+    max-width: 82%;
+    white-space: nowrap;
+    text-align: center;
+    color: var(--bubble-text);
+    font-family: "Trebuchet MS", "Verdana", sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Verdana" (font-family-name-quotes)

(font-family-name-quotes)